### ISSUE TITLE
Settings app crashes when changing keyboard shortcuts.

### ISSUE THREAD

**Issue by Justin00015** [pop-os/iso#163]
If this is a bug, please use the template below. If this is a feature request, question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os - as that is the proper forum for those types of posts.

**Reporter info**

```
 * Distribution - (run ```cat /etc/os-release```)
 * Related Application and/or Package Version - apt policy $PACKAGE NAME
```

**Issue/Bug Description**
When trying to change keyboard shortcuts in GNOME Settings, the app gets stuck and needs to be stopped.

**Steps to reproduce (if you know)**
Settings>Keyboard>Change any shortcut

**Expected behaviour**
A pop-up should appear telling you to input a new keyboard combination to change the shortcut.

**Other Notes**

**Comment by Appalbarry** [comment]
On my system the pop-up does appear telling you to input a new keyboard combination to change the shortcut.

However it is unresponsive (as is the "Activities" menu) and has be ESCed to close it.

barryrueger@pop-os:~$ cat /etc/os-release
NAME="Pop!_OS"
VERSION="17.10 (Artful Aardvark)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 17.10 (Artful Aardvark)"
VERSION_ID="17.10"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com/"
BUG_REPORT_URL="https://github.com/system76/pop-distro/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=artful
UBUNTU_CODENAME=artful

**Comment by brs17** [comment]
@Justin00015 @Appalbarry I have been unable to reproduce this bug with the latest version of Pop!_OS on an Oryx Pro 2 or a Galago 3. 

Are you two still having Settings crash on you when changing keyboard shortcuts?

**Comment by Appalbarry** [comment]
I see now that it does work.  Because I had never had a reason to set up a shortcut* it didn't immediately occur to me that it needed **Control Key + Letter key**.

When I tried it I only entered letter keys, which of course doesn't work.  This is one of those things that assumes specific knowledge that not every user will have.  An argument can be made that the dialog should tell users how they need to construct a shiortcut.

* OK, back in the days of WordPerfect 5.1 I was the king of the keyboard.  These days I don't seem to have the need.
